src/settings.platformsh.php

```diff
@@ -1,33 +1,33 @@
 <?php
-/**
- * @file
- * Platform.sh settings.
- */
 
-$platformsh = new \Platformsh\ConfigReader\Config();
+declare(strict_types = 1);
+
+use Platformsh\ConfigReader\Config;
```

Comment:
Let's turn phpcs for this file off, since we maybe want to diff it to updates in the future?